ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -383,7 +370,18 @@ function check_upgrade_status() {
             echo -e "Upgrade stuck at updating RHEL worker, need to run the RHEL worker upgrade later...\n\n"
             return 0
         fi
-        dump_status_if_unexpected
+        # assuming oc to equal TARGET_MINOR_VERSION, log abnormal "upgrade status" if available (version 4.16+)
+        if [[ -n "${TARGET_MINOR_VERSION}" ]] && [[ "${TARGET_MINOR_VERSION}" -ge "16" ]] ; then
+            # capture "update health" section of status, ignoring the title and the last line
+            # whenever an error is present, "Run with --details=health" is the last line we should remove.
+            # with no error, it'll remove "Update is proceeding well" resulting in empty $stat that we simply ignore
+            stat="$(env OC_ENABLE_CMD_UPGRADE_STATUS=true oc adm upgrade status 2>&1 | sed -ne "/= Update Health =/,$ p" | sed '1d;2d;$d' || true)"
+            # if update health messages exist, and the mesage is different from previous (ignoring time difference using $filter), dump "Update Health" to log
+            if [ -n "$stat" ] && ! diff -qw <(sed -zE "s/${filter}//g" <<< "${stat}") <(sed -zE "s/${filter}//g" <<< "${oldstat}") >/dev/null ; then
+                echo "${stat}"
```

Comment:
For better understanding where the message on the [log](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/53915/rehearse-53915-periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-stable-4.17-upgrade-from-stable-4.16-aws-ipi-proxy-sts-f28/1808142830237913088/artifacts/aws-ipi-proxy-sts-f28/cucushift-upgrade-toimage/build-log.txt) is from, shall we add some prompt message to indicated that before printing out them ?

Reply:
this one is a dilemma. on one side, having a title to each status message sounds like a more readable log, however on the other hand, status update health messages are easily recognizable because they are table structured as such
```
1h4m23s   Error     API Availability   Cluster Operator authentication is degraded (OAuthServerConfigObservation_Error)
6m37s     Warning   Update Stalled     Cluster Version version is failing to proceed with the update (ClusterOperatorDegraded)
-         Warning   Update Stalled     Outdated nodes in a paused pool 'mcpfoo' will not be updated
```
since it can be a number of lines, adding an inline title will look wrong, then adding a title on a separate line is an option, however if there's multiple error messages printed throughout the upgrade, that title will also appear multiple times, making the log longer to read, while providing no additional value.  
another idea however, since both "oc adm upgrade" and "upgrade status" are running one after the other at the same time, we can change instead of printing the current `Running command: oc adm upgrade` as generated by `run_command`, replace by something like : echo "Running oc adm upgrade and upgrade status (filtered)"